.circleci/config.yml

```diff
@@ -757,8 +767,6 @@ workflows:
       - build:
           name: build-local-pv-provisioner
           container-image: local-pv-provisioner
-          requires:
```

Comment:
The other components have no dependencies between `test` and `build` jobs.
So I removed the dependency for local-pv-provisioner as well.